### DISCUSSION TITLE
First Frigate+ Model = Flood of Misdetections - Is this Normal or Expected?

### DISCUSSION THREAD

**Discussion by domenicdistefano** [blakeblackshear/frigate#8069] [Ask A Question]
About a week ago I activated my Frigate+ subscription and started uploading & verifying images across my 12 outdoor cameras. After submitting ~650 images total with a good mix of confirmed good detections and invalid ones I decided to request my first model this morning. Misdetections were at a relatively low rate in general in both day and night conditions. I had found that my misdetections from day to day were consistently in the same areas and similar. My expectation was that after the model was completed and installed my misdetections would go down to near zero for at least the areas / scenarios  I had flagged. I made sure to upload as many of the same / identical misdetections as I would find figuring more was better for the model during analysis.

This evening (well after nightfall - all cameras now in IR mode) I was notified my model was ready so I entered the ID in my config & restarted.

**Things is where everything went nuts**. Within the first couple of minutes I started getting dozens & dozens of misdetections across three or so cameras of people & dogs. Zero of these actually contained any people, animals, or anything except light shining on something stationary or static leaves on a tree. Also none of these were ever false detections before activating the model. I suspect by morning I will have several hundred of these false detections. Again they are very repetitive of the same tree branch for instance in dozens of detections. Will be interesting to see what happens in the daylight.

So what should I do? Do I wait for morning and upload some # or sample of the identical false positives or god forbid all of them? Then at some point kick off another model run to hopefully weed things out? If so how long or how many samples should I upload per bad detection that is identical?

Thanks

**Comment by NickM-27** [comment]
Multiple users have reported that the first model had higher false positives and the second model got much better. 

https://github.com/blakeblackshear/frigate/discussions/7932#discussioncomment-7189445

https://github.com/blakeblackshear/frigate/discussions/7932#discussioncomment-7166293

This is also called out in the docs. Have you updated your threshold and min score like the docs recommend?

https://deploy-preview-6262--frigate-docs.netlify.app/plus/#higher-scores

**Reply by domenicdistefano** [reply]
Looks like what am seeing is normalish  - very good! I had been upping my thresholds incrementally already trying to reduce false positives but a bit below the suggestions. I will adjust that now and see what happens. My false positives so far have been >85% to low 90s.

~ how many identical samples of a false positive in the same place should I upload? Just a few or many or all?

**Reply by NickM-27** [reply]
It's important to have a good balance of true and false positives is the main thing. Definitely recommend submitting a good amount overall
